src/Core/Result.php

```diff
@@ -76,6 +73,32 @@ protected function populateFromResponse(ResponseInterface $response): void
         }
     }
 
+    /**
+     * Returns info on the current request
+     *
+     * @return array{
+     *   resolved: bool
+     *   response?: ?ResponseInterface,
+     *   status?: int
+     * }
+     */
+    final public function info(): array
+    {
+        if (!isset($this->response)) {
+            return [
+                'resolved' => true,
+            ];
+        }
+
+        $info = $this->response->getInfo();
```

Comment:
```suggestion
        $httpStatus = $this->response->getInfo('http_code');
```

Reply:
Cool. I didn't know that shortcut.

Don't be mad at me, I didn't pick your suggestio because didn't used the intermediate variables.

Reply:
Oh. I’m not mad at all. 
I’m happy you are delivering so much excellence. :)